File: src/main/java/com/swp391/koibe/repositories/OrderDetailRepository.java
package com.swp391.koibe.repositories;

import com.swp391.koibe.models.OrderDetail;
import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;

public interface OrderDetailRepository extends JpaRepository<OrderDetail, Long> {

    List<OrderDetail> findByOrderId(Long orderId);

}

File: src/main/java/com/swp391/koibe/repositories/OrderDetailRepository.kt
package com.swp391.koibe.repositories

import com.swp391.koibe.models.OrderDetail
import org.springframework.data.jpa.repository.JpaRepository

interface OrderDetailRepository : JpaRepository<OrderDetail?, Long?> {
    fun findByOrderId(orderId: Long?): List<OrderDetail?>?
}
